hummingbot/client/command/connect_command.py

```diff
@@ -55,6 +55,7 @@ async def connect_exchange(self,  # type: HummingbotApplication
                     return
                 Security.update_secure_config(config.key, config.value)
             api_keys = await Security.api_keys(exchange)
+            api_keys[config.key] = config.value
```

Comment:
why does this last api key (last config from for loop) need to be assigned value explicitly?